<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HallWidget</class>
 <widget class="QWidget" name="HallWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButtonUserList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton {
	font: 25 12pt &quot;Microsoft YaHei UI Light&quot;;
    background-color: rgb(239, 180, 121);
   border-color: rgb(16, 16, 16);
	border: 2px solid rgb(0,0,0);
	border-radius : 35px;
}

QPushButton:hover {
   background-color: khaki;
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: #d0d67c;
}
</string>
    </property>
    <property name="text">
     <string>用户列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFight">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton {
	font: 25 12pt &quot;Microsoft YaHei UI Light&quot;;
    background-color: rgb(239, 180, 121);
   border-color: rgb(16, 16, 16);
	border: 2px solid rgb(0,0,0);
	border-radius : 35px;
}

QPushButton:hover {
   background-color: khaki;
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: #d0d67c;
}
</string>
    </property>
    <property name="text">
     <string>战斗大厅</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPack">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton {
	font: 25 12pt &quot;Microsoft YaHei UI Light&quot;;
    background-color: rgb(239, 180, 121);
   border-color: rgb(16, 16, 16);
	border: 2px solid rgb(0,0,0);
	border-radius : 35px;
}

QPushButton:hover {
   background-color: khaki;
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: #d0d67c;
}
</string>
    </property>
    <property name="text">
     <string>查看背包</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonQuit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton {
	font: 25 12pt &quot;Microsoft YaHei UI Light&quot;;
    background-color: rgb(239, 180, 121);
   border-color: rgb(16, 16, 16);
	border: 2px solid rgb(0,0,0);
	border-radius : 35px;
}

QPushButton:hover {
   background-color: khaki;
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: #d0d67c;
}
</string>
    </property>
    <property name="text">
     <string>退出登录</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
